canvasapi/discussion_topic.py

```diff
@@ -1,3 +1,5 @@
+import re
```

Comment:
This change might have tagged along from a different commit.

Reply:
😖
I should've done a clean pull...I rebased instead of squashing and merging. Sorry.